SCClassLibrary/Common/Quarks/Git.sc

```diff
@@ -34,6 +34,44 @@ Git {
 			url = this.remote;
 		}
 	}
+
+	log { |limit ...args|
+		var format, cmd, log, formatCodes;
+		formatCodes = [
+			["%H", 	\commit_hash],
+			["%T", 	\tree_hash],
+			["%an",	\author_name],
+			["%ae",	\author_email],
+			["%ad",	\date],
+			["%s",	\subject],
+			["%b",	\body],
+		].flop;
+
+		format = formatCodes[0].join("%x1f") ++ "%x1e";
+		cmd = ["log", "--format='%'".format(format), "--date=short"];
+		if (limit.notNil) {
+			cmd = cmd ++ ["-%".format(limit.asInteger.asString)];
```

Comment:
`asString` is unnecessary here (in fact, the simplest way to write it IMO would be `"-" ++ limit.asInteger`